.github/issue_template.md

```diff
@@ -1,22 +1,45 @@
-Please read this text and delete everything up to the line of dashes before
-filing your issue. Please
+<!--
+Thank you for opening issue.
+Please read this text and provide all necessary info. Please understand that without
+reproduce steps, clear explanations, and information about your particular setup 
+issue may be closed as unreproducible.
+Lines enclosed by arrows are comments, and will be deleted automatically.
+-->
 
-- fill out the table below the line of dashes and keep it at the end of your
-  issue text. The provided answers below are examples, only keep one of them or
-  replace them entirely with something else. But please provide an answer for
-  every line.
+<!-- Issue template:
+- First please provide clear explanations of issue type. What causing it,
+  and how we could test it.
+- Provide short and understandable steps to reproduce the issue in the list
+  below.
+- Fill out the table below ## symbols and keep it at the end of your
+  issue text. Please provide an answer for every line.
 - provide a minimal viable repro case, ideally following
   https://github.com/SirVer/ultisnips/blob/master/CONTRIBUTING.md#reproducing-bugs.
 - If this is not possible, post a minimal, complete `.vimrc`, snippet
   definition, and set of key strokes that reproduces your problem.
-- remember to delete this boilerplate text up to the line of dashes.
+-->
 
----------------------
+<!-- Please post your explanations below -->
 
-- **OS**:  e.g. Windows XP / Ubuntu Linux / Mac OS 10.5
-- **Vim**: e.g. Vim 8.0, Vim 7.4, Neovim 0.3
-- **UltiSnips**: e.g. 3.1, commit 16c999e8c71134401a78d4d46435517b2271d6ac. Do
-  not write `latest master`, please look up the actual revision.
-- **Python inside Vim**: e.g. 2.7.14 / 3.6.5. If unsure run 
+
+<!-- Please provide steps to reproduce your issue.
+It may be complete instructions, set of key strokes, type of snippet, options etc. 
+You are free to extend list if needed
+-->
+**Steps to reproduce**
+
+1.
+2.
+3.
+
+
+## 
+<!-- Please fill theese fields -->
+- **Operating System**:  <!-- e.g. Windows XP / Ubuntu Linux / Mac OS 10.5 -->
```

Comment:
I think this has the danger that people will write their answer inside the comment block. That is not terrible though, since we can edit the bug they submit and change the lines.